README.md

```diff
@@ -68,3 +73,48 @@ You can find the greentea-client example application [here](./example/main.cpp).
     cmake --build cmake_build
 
 This generates an executable `./cmake_build/example/greentea-client-example`.
+
+# Adding greentea-client to a project
```

Comment:
@Patater @rwalton-arm Please review whether this section is consistent with how we integrate greentea-client with mbed-os, thanks.

Reply:
One small issue that I can see: CMake isn't a "build system", it's a build system generator. The "build system" would be Ninja, make etc.